lib/ansible/modules/cloud/azure/azure_rm_deployment.py

```diff
@@ -496,6 +497,11 @@ def deploy_template(self):
                 uri=self.template_link
             )
 
+        if self.append_tags and self.tags:
```

Comment:
append_tags is always None at this moment ?

Reply:
Not if you pass the `append_tags` parameter to the module.
cf. https://github.com/ansible/ansible/blob/5553b208281d723ead1e61be5c37227ec2b2736e/lib/ansible/module_utils/azure_rm_common.py#L54-L57

Reply:
parameters  tags nor append_tags  are not documented/listed in the module.

But that snippet says the default is true so i should not have to use that parameter, right ?

Reply:
Right. This parameter is common to all azure modules.